estate/models/estate_property.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from odoo.tools import relativedelta
+
+def _compute_three_months_from_now():
+    return fields.Date.today() + relativedelta(months=3)
+
+class EstateProperty(models.Model):
+
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    active = fields.Boolean(default=True)
+    state = fields.Selection([
+        ('new', 'New'),
+        ('offer_received', 'Offer Received'),
+        ('offer_accepted', 'Offer Accepted'),
+        ('sold', 'Sold'),
+        ('canceled', 'Canceled')
+    ], default='new', required=True, copy=False)
+
+    postcode = fields.Char()
+    date_availability = fields.Date(default=_compute_three_months_from_now(), copy=False)
```

Comment:
`compute_xxx`methods have a specific meaning in Odoo. They should only be used for computed fields. You have two options regarding the default property:

1. Use a lambda expression
2. Rename the `_compute_three_months_from_now` => `_get_default_date_availability`